web/src/content/posts/rsc-now-in-redwoodjs.mdx

```diff
@@ -0,0 +1,678 @@
+Welcome to the preview of RedwoodJS Bighorn! You may not have realized it, but you’ve been living in the Arapahoe epoch since Redwood v1.0. Bighorn is the next epoch, and will bring a massive change to how apps are built with Redwood: [React Server Components](https://www.joshwcomeau.com/react/server-components/) (RSC) and [Server-side Rendering](https://community.redwoodjs.com/t/react-streaming-and-server-side-rendering-ssr/5052) (SSR)!
+
+Core team lead Tobbe [announced](https://community.redwoodjs.com/t/react-server-components-rsc/5081) the first RSC build back in July of 2023 and the team has been hard at work on it ever since. We’re ready to show everyone what we’ve been working on and invite you to try it out yourself and give us feedback! This document walks you through what’s different about a Redwood RSC app versus a “classic” Redwood app built with GraphQL. We’re going to review an example app that we’ll convert from GraphQL to RSC, demonstrating the surprisingly small number of steps required to do so.
+
+<div data-node-type="callout">
+<div data-node-type="callout-emoji">💡</div>
+<div data-node-type="callout-text">Redwood’s epochs are named after <a target="_blank" rel="noopener noreferrer nofollow" href="https://en.wikipedia.org/wiki/List_of_national_forests_of_the_United_States" style={{pointerEvents: 'none'}}>National Forests</a> in the United States!</div>
+</div>
+
+You can also watch a video covering how RSC works in Redwood and how to convert an existing GraphQL app:
+
+<iframe className="w-full aspect-video border-0" src="https://www.youtube.com/embed/5IZv3khsx0o?rel=0&modestbranding=1" title="YouTube video player" allowFullScreen></iframe>
```

Comment:
I don't know why but this started not to have the correct aspect ratio applied. The video was too short. 

It also doesn't have the same margin at the bottom as it used to. Again sorry @ahaywood do you mind taking a look at it?